### PR TITLE
GhostBoxGeometryVariableFillPattern, to ignore the border node (overwrite_interior) for copy/stream operations for same level overlaps

### PR DESCRIPTION
**Pull request by PhilipDeegan**

### CODE REVIEW

File: src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+
+#include "GhostBoxGeometryVariableFillPattern.h"
```
Comment (nicolasaunai) [inline src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp:3]: maybe more explicit if renamed into `GhostOnlyVariablyFillPattern`

File: src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+namespace SAMRAI::xfer
+{
+const std::string GhostBoxGeometryVariableFillPattern::s_name_id = "BOX_GEOMETRY_FILL_PATTERN";
```
Comment (nicolasaunai) [inline src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp:7]: don't we need to change that string?

File: src/amr/messengers/GhostBoxGeometryVariableFillPattern.h [hunk 3]
```diff
@@ -0,0 +1,178 @@
+
+#ifndef GhostBoxGeometryVariableFillPattern_
+#define GhostBoxGeometryVariableFillPattern_
+
+#include "SAMRAI/xfer/VariableFillPattern.h"
+
+namespace SAMRAI::xfer
```
Comment (nicolasaunai) [inline src/amr/messengers/GhostBoxGeometryVariableFillPattern.h:7]: should we keep these namespaces?

File: src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp (hunk 2 above)
Comment (PhilipDeegan) [reply src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp:7]: should

Comment (PhilipDeegan) [reply src/amr/messengers/GhostBoxGeometryVariableFillPattern.cpp]: should, but can't

File: src/amr/messengers/GhostBoxGeometryVariableFillPattern.h (hunk 3 above)
Comment (PhilipDeegan) [reply src/amr/messengers/GhostBoxGeometryVariableFillPattern.h]: dropped

**Comment by nicolasaunai** [comment]
The code in this PR prevents schedules from exchanging primal shared border nodes.

**Comment by PhilipDeegan** [comment]
obsoleted by #576

**Pull request closed without merging**